usr.sbin/ppp/ppp.8

```diff
@@ -2166,13 +2166,15 @@ set device /dev/cuau0
 set speed 115200
 .Ed
 .Pp
-Cuad0 is the first serial port on
+.Pa cuau0
+is the first serial port on
 .Fx .
```

Comment:
Yes and no. cuau0 is the first serial port that's connected to ACPI, ISA or PCI bus via one of the uart attachments FreeBSD has. This is true on all our tier1 platforms, in the absence of wiring or other issues.
I'd be mildly tempted to say that cuaU0 is the first USB serial port, but I'm not sure how to work that in, exactly.
ppp(8) was written when there were no USB attached serial ports to note (and none good enough to run PPP). Now, you're more likely to have a USB serial port than a hard wired one (modems, what are those :).

Reply:
Yes, because it can be confusing.
Device naming in manual page changelog(-ish):

| Name | Introduced | Manual Page Fix |
| ---- | ---------- | ----- |
| `cua00` (SIO) | af57ed9fdcc4e2a81940fc2243c0d3949fc0e00f | - |
| `cuaa0` | ab9ea4a7256ce2e1289bd82a31ec5d210a9f21aa | c853fdb82db495d0ec307f1f81a9003d670747b0 |
| `cuad0` | 9cb9afbd68d2088291ec4017cf48dde099b68582 | 37c510c7c0c46b344026d7b0eb65d0cf91420c39 |
| `cuau0` (UART) | ae59037978535fd06e84ab87a298fc54ee084bbb | this PR[^1] |


[^1]: Missed in 839f11a4fe18e4ae2dd930766b551fa67e354735.

Reply:
This feels a little rough, but it is a little rough already:
```
.Pa cuaU0
and
.Pa cuau0
are the first
.Xr usb 4
and
.Xr uart 4
serial ports found by
.Fx ,
respectively.
```

Reply:
Good! thank you for the input.